### PR TITLE
Fix args

### PR DESCRIPTION
**Pull request by riita10069**
# Description

Fixes # (issue)

## Type of change

- [ ] Document update or simple typo fix
- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

- [ ] I have checked backward/forward compatibility that may cause regarding this change.

### CODE REVIEW

File: manifests/infra/mysql-o11y/overlays/development/cronjob.yaml [hunk 1]
```diff
@@ -10,6 +10,8 @@ spec:
         spec:
           containers:
           - name: mysql
+            image: mysql:8.0
+            args: ["mysqldump --single-transaction --quick --set-charset -u $RDS_USER -p $RDS_PASSWORD -h $RDS_HOST dreamkast conference_days conferences proposal_item_configs proposal_items proposals talk_categories talk_difficulties talk_times talks talks_speakers tracks stats_of_registrants viewer_counts chat_messages > $HOME/rds.dump; mysql -u $MYSQL_USER -p $MYSQL_PASSWORD  -h $MYSQL_HOST dreamkast < $HOME/rds.dump"]
```
Comment (inductor) [inline manifests/infra/mysql-o11y/overlays/development/cronjob.yaml]: ここはいらないようにも見えるけど必要な差分ですか？

**Review by inductor — APPROVED** [review]

File: manifests/infra/mysql-o11y/base/cronjob.yaml [hunk 2]
```diff
@@ -12,7 +12,46 @@ spec:
...
+            - proposal_items
+            - proposals
+            - talk_categories
+            - talk_difficulties
+            - talk_times
+            - talks
+            - talks_speakers
+            - tracks stats_of_registrants
+            - viewer_counts
+            - chat_messages
```
Comment (inductor) [inline manifests/infra/mysql-o11y/base/cronjob.yaml:42]: chat message いらないんじゃなかったっけ

**Review by inductor — CHANGES_REQUESTED** [review]

File: manifests/infra/mysql-o11y/base/cronjob.yaml (hunk 2 above)
Comment (riita10069) [reply manifests/infra/mysql-o11y/base/cronjob.yaml:42]: 本当だ。
鋭いですね

**Comment by inductor** [comment]
rebaseのほうがコミットログきれいになりそうだけどおまかせします

File: manifests/infra/mysql-o11y/base/cronjob.yaml [hunk 3]
```diff
@@ -12,7 +12,45 @@ spec:
...
+            - conferences
+            - proposal_item_configs
+            - proposal_items
+            - proposals
+            - talk_categories
+            - talk_difficulties
+            - talk_times
+            - talks
+            - talks_speakers
+            - tracks stats_of_registrants
```
Comment (inductor) [inline manifests/infra/mysql-o11y/base/cronjob.yaml:40]:
```suggestion
            - tracks
            - stats_of_registrants
```

**Review by inductor — CHANGES_REQUESTED** [review]

**Review by inductor — APPROVED** [review]
やっていくか

**Pull request merged**